Library/Formula/apache-spark.rb

```diff
@@ -4,10 +4,10 @@ class ApacheSpark < Formula
   desc "Engine for large-scale data processing"
   homepage "https://spark.apache.org/"
   head "https://github.com/apache/spark.git"
-  url "https://d3kbcqa49mib13.cloudfront.net/spark-1.3.1-bin-hadoop2.6.tgz"
-  version "1.3.1"
+  url "https://d3kbcqa49mib13.cloudfront.net/spark-1.4.0-bin-hadoop2.6.tgz"
```

Comment:
Please use the canonical Apache URL here:
https://www.apache.org/dyn/closer.cgi?path=spark/spark-1.4.0/spark-1.4.0-bin-hadoop2.6.tgz

(sorry for missing this)